DataConnectors/ThycoticCEF.json

```diff
@@ -0,0 +1,115 @@
+{
+    "id": "ThycoticCEF",
+    "title": "Thycotic Secret Server",
+    "publisher": "Thycotic, Inc",
+    "descriptionMarkdown": "Common Event Format (CEF) from Thycotic Secret Server ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Thycotic Secret Server",
```

Comment:
Legend should be **_CommonSecurityLog_** , the data type name, Please refer CEF templatehttps://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json